LDAR_Sim/src/default_parameters/p_default.yml

```diff
@@ -24,6 +24,8 @@ emissions:
   subtype_leak_dist_file: "_placeholder_str_"
   units: ["kilogram", "hour"]
 NRd: 365
+n_init_leaks: "_placeholder_int_"
+n_init_days: "_placeholder_int_"
```

Comment:
[Nit] n_init_leaks is a great name and very clearly conveys the purpose of the parameter. Do you feel n_init_days does the same?

Reply:
I believe so, it also keeps the same nomenclature as n_init_leaks, to help users understand that it affects a similar place in the simulation.